CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+We accept contributions via Pull Requests on [GitHub](https://github.com/Yoast/YoastSEO.js).
+
+## Pull Requests
+
+- **Coding standards** - The easiest way to verify the coding standards is to use `grunt eslint`.
+
+- **Add tests!** - Your patch won't be accepted if it doesn't have tests.
```

Comment:
Let's keep this friendly and focus on growing our community: "Your patch will be merged sooner if it has tests."